packages/webpack-plugin/package.json

```diff
@@ -17,19 +17,25 @@
     "./extracted": {
       "module": "./extracted/dist/vanilla-extract-webpack-plugin-extracted.esm.js",
       "default": "./extracted/dist/vanilla-extract-webpack-plugin-extracted.cjs.js"
+    },
```

Comment:
Wondering if this is the [desired place](https://github.com/seek-oss/vanilla-extract/issues/374#issuecomment-1060046223) for having the virtual loader. I took a best guess to start out with.

Reply:
I agree it should live here but could we name it something a bit clearer. How about `virtualFileLoader`?